projects/valtimo/components/src/lib/components/carbon-list/carbon-list.component.html

```diff
@@ -107,3 +107,14 @@
     </cds-overflow-menu-option>
   </cds-overflow-menu>
 </ng-template>
+
+<ng-template #rowDisabled let-data="data">
+  <div *ngIf="data.locked" class="disabled">
+    <span
+      class="float-right badge badge-pill badge-secondary bg-grey"
+      ngbTooltip="{{ translationKey + '.rowLocked' | translate }}"
```

Comment:
Leave this just as a 'lockTooltip' variable instead of doing translation in the table. If it's done like this, then we limit implementation teams in the way that they structure their translations

Reply:
Changed